megabeast/tools/make_naive_maps.py

```diff
@@ -83,7 +83,7 @@ def create_naive_maps(stats_filename, pix_size=10.0):
             # tindxs, = np.where((x == i) & (y == j) & (cat['chi2min'] < 10.))
             if len(tindxs) > 0:
                 summary_stats[j, i, n_sum] = len(tindxs)
-                print(i, j, len(tindxs))
+                # print(i, j, len(tindxs))
```

Comment:
Anyway to make this optional instead if just commenting it out?  As you say - good for checks.  Maybe add a verbose keyword and have it default to False?

Reply:
Makes sense. Do you mean adding a keyword as in, say, [model_grid.py](https://github.com/BEAST-Fitting/beast/blob/master/beast/physicsmodel/model_grid.py), for a print/no print? I can def do this.